src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -102,12 +102,12 @@ impl ScalarFunc {
         ctx: &mut EvalContext,
         row: &'a [Datum],
     ) -> Result<Option<Cow<'a, [u8]>>> {
-        let t = try_opt!(self.children[0].eval_time(ctx, row));
+        let t: Cow<'a, Time> = try_opt!(self.children[0].eval_time(ctx, row));
         let month = t.get_time().month() as usize;
         if t.is_zero() && ctx.cfg.mode_no_zero_date_mode() {
             return handle_incorrect_datetime_error(ctx, t).map(|_| None);
         } else if month == 0 || t.is_zero() {
-            return Ok(Some(Cow::Owned("".to_string().into_bytes())));
+            return Ok(None);
```

Comment:
Any test cover it?

Reply:
Related test cases have just been added. PTAL again.